### PR TITLE
Pin openshift ansible in ci

### PR DESCRIPTION
**Pull request by tomassedovic**
#### What does this PR do?
This pins a specific openshift-ansible version in the openstack end to end CI that sholudn't be broken for us.

#### How should this be manually tested?
The end to end ci should pass. No manual testing necessary.

#### Is there a relevant Issue open for this?
Issue #686 

#### Who would you like to review this?
cc: @Tlacenka @tzumainn  @bogdando PTAL

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ python:
 env:
   global:
     - CI_CONCURRENT_JOBS=1
-    - OPENSHIFT_ANSIBLE_COMMIT=master
+    - OPENSHIFT_ANSIBLE_COMMIT=1febf0b20de622d358dc890417dc4cda7c3f71a9
```
Comment (bogdando) [inline .travis.yml]: please use a tag?

Comment (tomassedovic) [inline .travis.yml]: The CI's running now so I'll let it run its course. We'd have to re-run it with the tag afterwards (to make sure that fixes it) so if you don't mind, I'd just leave this one be and I'll make sure to use tags here in the future.

**Comment by tomassedovic** [comment]
@bogdando the CI passed, can we merge this to unblock everyone or do you want me to switch to using tags?

**Comment by Tlacenka** [comment]
LGTM :)

**Pull request merged**